exe/puppet-db.cc

```diff
@@ -40,6 +41,17 @@ main(int argc, char **argv) {
         po::options_description global_options("global options");
         global_options.add_options()
                 ("help,h", "produce help message")
+                ("config,c", po::value<string>()->default_value(
+                    "~/.puppetlabs/client-tools/puppetdb.conf"),
+                 "path to use for cli configuration")
+                ("urls", po::value<string>()->default_value(""),
+                 "urls, `http://foo:8080,http://bar:8080`, to use for puppetdb")
```

Comment:
How about 'list of urls for connecting to puppetdb, urls can be comma separated. I don't know if we have a better place for examples :-).